src/DripsHub.sol

```diff
@@ -219,21 +222,24 @@ abstract contract DripsHub {
         view
         returns (uint128 collected, uint128 split)
     {
-        ReceiverState storage receiver = _dripsHubStorage().receiverStates[user];
+        bytes32 userHash = _hashUser(user);
+        DripsHubStorage storage dripsHubStorage = _dripsHubStorage();
+        DripsState storage dripsState = dripsHubStorage.dripsStates[userHash][DUMMY_ASSET];
+
         _assertCurrSplits(user, currReceivers);
 
         // Collectable independently from cycles
-        collected = receiver.collectable;
+        collected = dripsHubStorage.splitsStates[userHash].collectables[DUMMY_ASSET];
```

Comment:
I think it would be cleaner
```solidity
uint collected = 0;
// collect logic
collected += dripsHubStorage.splitsStates[userHash].collectables[DUMMY_ASSET]
// split logic
```

Reply:
`collected` is a returned variable, so it's already initialized to 0, writing `collected = 0;` to me just doesn't feel right. Would only turning `=` into `+=` be clear enough?

```solidity
collected += dripsHubStorage.splitsStates[userHash].collectables[DUMMY_ASSET];
```